client/src/pages/CreateEnergiser.js

```diff
@@ -0,0 +1,96 @@
+import { useState } from "react";
+import Navbar from "./components/Navbar";
+import Footer from "./components/Footer";
+import "./CreateEnergiser.css";
+
+
+ const CreateEnergiser = () =>{
```

Comment:
Indentation here is a bit confusing, all lines below this one should be intended (white space)